csfieldguide/templates/interactives/unicode-binary.html

```diff
@@ -4,7 +4,7 @@
 {% load static %}
 
 {% block html %}
-  <div data-iframe-height>
+  <div data-iframe-height class="container-fluid">
```

Comment:
Are we able to get more padding? Might affect how it looks in-page though..

![image](https://user-images.githubusercontent.com/16066822/57894871-c103e500-789c-11e9-8296-91a84995e839.png)